src/middlewared/middlewared/api/v25_04_0/smartctl.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any
+
+from middlewared.api.base import BaseModel
+
+__all__ = ["AtaSelfTest", "NvmeSelfTest", "ScsiSelfTest"]
+
+class AtaSelfTest(BaseModel):
```

Comment:
flake8 does not complain here, because new error count is less than previous error count, but every class declaration should be prefixed by two newlines.